wsgi.py

```diff
@@ -58,7 +59,11 @@ def post_solver_result():  # Ignore PyDocStyleBear
 
 @application.route('/api/v1/adviser-result', methods=['POST'])
 def post_adviser_result():  # Ignore PyDocStyleBear
-    return jsonify({'error': 'Not implemented yet'}), 500, {'ContentType': 'application/json'}
+    adapter = AdvisersResultsStore()
+    adapter.connect()
```

Comment:
No check if we are connected?

Reply:
The main flow is that connection should succeed and results are stored. If connection or storing results fails, there is raised an exception that will be propagated back to producer as an HTTP error and we will see it in the logs on both sides.